ecommerce/CartController.php

```diff
@@ -189,36 +198,19 @@ public function tutor_delete_course_from_cart() {
 			wp_send_json_error( tutor_utils()->error_message( 'nonce' ) );
 		}
 
+		$user_id   = tutils()->get_user_id();
 		$course_id = Input::post( 'course_id' );
```

Comment:
We don't need to use is_numeric we can leverage TYPE_INT in the following way:
`$course_id = Input::post( 'course_id' , 0, Input::TYPE_INT );` 

Then can just do: `if ( ! $course_id )`